frontend/src/metabase/querying/notebook/components/NotebookDataPicker/EmbeddingDataPicker/tests/EmbeddingDataPicker.unit.spec.tsx

```diff
@@ -1,10 +1,15 @@
 import userEvent from "@testing-library/user-event";
 
+import { setupEnterprisePlugins } from "__support__/enterprise";
 import { screen } from "__support__/ui";
 
 import { setup } from "./setup";
 
 describe("EmbeddingDataPicker", () => {
+  beforeEach(() => {
```

Comment:
Since existing tests tested multi-stage data picker, and it wasn't an EE plugin before, we didn't need to set up any EE plugins. But now that the multi-stage data picker is in an EE plugin, we need to initialize it.